src/components/MenuItem.tsx

```diff
@@ -153,9 +153,15 @@ type MenuItemBaseProps = {
     /** Error to display at the bottom of the component */
     errorText?: MaybePhraseKey;
 
+    /** Any additional styles to pass to error text. */
+    errorTextStyle?: StyleProp<ViewStyle>;
+
     /** Hint to display at the bottom of the component */
     hintText?: MaybePhraseKey;
 
+    /** Should the error text red dot indicator be shown */
+    shouldShowErrorTextRedDot?: boolean;
```

Comment:
Why not name this as `shouldShowRedDotIndicator`? This is consistent across and red dot is generally associated with errors.